gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureRunnerResource.java

```diff
@@ -111,13 +92,12 @@ static ConjureRunner createNewRunner(File executable) throws IOException {
                                     MemberSubstitution.relaxed()
                                             .method(ElementMatchers.is(System.class.getMethod("exit", int.class)))
                                             .replaceWith(GradleExecStubs.getStubMethod())))
-                    .make(typePool)
-                    .load(classLoader, ClassLoadingStrategy.Default.INJECTION)
+                    .make()
+                    .load(Thread.currentThread().getContextClassLoader(), ClassLoadingStrategy.Default.INJECTION)
```

Comment:
Is this loading it into the current classloader? Won't that mean if this method is called multiple times, we'll load the same patched class multiple times? If so, does this work with main method in the gradle configuration classloader and the generator code in the worker classloader.

We also have a `classLoader` parameter on this method, whose argument is only ever `Thread.currentThread().getContextClassLoader()` - should we use that instead?